pkg/api/souin.go

```diff
@@ -93,9 +93,15 @@ func (s *SouinAPI) BulkDelete(key string, purge bool) {
 }
 
 // Delete will delete a record into the provider cache system and will update the Souin API if enabled
+// The key can be a regexp to delete multiple items
 func (s *SouinAPI) Delete(key string) {
 	for _, current := range s.storers {
-		current.Delete(key)
+		_, err := regexp.Compile(key)
```

Comment:
`Compile` once outside the `for` loop should be enough, the `key` value won't change between two storers.

Reply:
Oh yeah, thanks for your suggestion!